design/one-pager-mrds.md

```diff
@@ -0,0 +1,248 @@
+# Managed Resource Definitions
+
+* Owner: Nic Cope (@negz)
+* Reviewer: ???
+* Status: Draft
+
+## Background
+
+Crossplane managed resources (MRs) suffer from two long-standing issues.
+
+MRs can automatically write sensitive connection details like addresses,
+usernames, and passwords to a Kubernetes Secret. We call these connection
+details. Today it's very hard to discover what connection details an MR
+supports. The only way to do so is to create an MR and see what it writes to its
+connection secret. This is tracked in [issue 1143][1] - Crossplane's third most
+upvoted issue.
+
+Crossplane bundles related MRs together into a provider - like provider-github
+or provider-aws-ec2. Today some of these providers install up to 100 MRs. With
+v2 this'll double, because v2 introduces namespaced MRs but keeps the existing
+cluster scoped MRs for backward compatibility.
+
+Each MR is powered by a CustomResourceDefinition (CRD) and a controller. Each
+CRD has a performance penalty on the Kubernetes API server - mostly memory
+usage. Each CRD also creates one or more API endpoints that clients like kubectl
+may need to walk.
+
+It's not possible to install only some of a provider's MRs. It's all or nothing.
```

Comment:
Why not have a spec for a list of resources we would like to install from a provider as part of the provider spec instead?

Reply:
Covered in the `spec.enablesAPIs` section under alternatives considered - I think you commented there too so I'll resolve this and discuss there.